aqua-examples/ts-oracle/Readme.md

```diff
@@ -106,84 +161,140 @@ data Oracle:
     raw_data: []u64
 
 -- the point_estimate function from the Wasm file wrapped as a service
--- marine aqua artifacts/ts_oracle.wasm
+-- marine aqua ts-oracle/modules/ts-oracle/target/wasm32-wasi/release/ts_oracle.wasm
```

Comment:
why are the wasm services no moved from the artifacts dir? every time `cargo clean` is run, the wasm files are gone.

Reply:
Changed that since the CLI doesn't support build customization yet. I agree with artifacts it's convenient